code/modules/admin/admin_verbs.dm

```diff
@@ -1212,21 +1212,21 @@ var/list/fun_images = list()
 	var/rendered = "<span class='game hivesay'><span class='prefix'>HIVEMIND:</span> <span class='name'>ADMIN([show_other_key ? src.fakekey : src.key])</span> says, <span class='message'>\"[msg]\"</span></span>"
 	var/adminrendered = "<span class='game hivesay'><span class='prefix'>HIVEMIND:</span> <span class='name' data-ctx='\ref[src.mob.mind]'>[show_other_key ? "ADMIN([src.key] (as [src.fakekey])" : "ADMIN([src.key]"])</span> says, <span class='message'>\"[msg]\"</span></span>"
 
-	for (var/mob/M in mobs)
+	for (var/client/C in clients)
```

Comment:
cheaper to loop through clients than to loop through mobs and check if they have a client